### PR TITLE
JS-2-W1 Assignment Completed {Codelawer}

### PR DESCRIPTION
**Pull request by codelawer**
This repo forked, and cloned via using gitbash.
The following steps had been taken in **gitbash** : 

1. - git add .; 
2. - git commit -m, 
3. - git push,

In browser send 

- [ ] pull request.

### CODE REVIEW

File: Week1/exercise/w1/index.js [hunk 1]
```diff
@@ -28,3 +44,13 @@ console.log('header: ', header);
 //    create <li> element
 //    set <li> element innerHtml to todoInput value
 //    add <li> element to todoList
+
+const addTodoButton = document.getElementById('btn-addTodo');
+const addTodo = () => {
+  const getTodoList = document.querySelector('#todoList');
```
Comment (bitsagarob) [inline Week1/exercise/w1/index.js]: better to define these variables outside this function such that they are only selected once and not every time this function is called

**Review by bitsagarob — CHANGES_REQUESTED** [review]

File: Week1/exercise/w1/index.js [hunk 2]
```diff
@@ -15,7 +10,28 @@ console.log('header: ', header);
 //    set image src to imageInput element value
 
 // ====================================== //
+const ChangeImageButton = document.querySelector('#btn-changeImage');
+console.log(ChangeImageButton);
+const changeImage = () => {
+  const imgElement = document.querySelector('#imageToChange');
```
Comment (bitsagarob) [inline Week1/exercise/w1/index.js]: better to define these variables outside this function such that they are only selected once and not every time this function is called

**Review by bitsagarob — CHANGES_REQUESTED** [review]

**Review by ldealmei — CHANGES_REQUESTED** [review]
Well done for the first exercice.
For the second though, have you tested it? Does it behave as we would expect a to-do list to work?
What can you do to improve on the current behavior?
Hint: we often have to-do lists for more than one thing

**Comment by ldealmei** [comment]
Exercices 1 & 2 are good :) 
Have a try for the book exercise!

File: Week1/homework/main.js [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+      li.setAttribute("id", keys[i])
+      ul.appendChild(li);
+      const h1 = document.createElement("h1");
+      const removeUnderscores = keys[i].replace(/_/g, " ");
+      h1.innerText = removeUnderscores;
+      li.appendChild(h1);
+      h1.style.color="#000"
+      h1.style.textTransform="uppercase";
+
+      const h2 = document.createElement("h2");
```
Comment (bitsagarob) [inline Week1/homework/main.js:183]: these h2, h3, h4 can have better naming. Think in what they represent, not what they are

File: Week1/homework/main.js [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+  }
+  createElement()
+
+  //additional things
+
+  const newButton = document.createElement('button');
+  const rootOfMain = document.querySelector('.wrapper').appendChild(newButton);
+  newButton.setAttribute('id', 'see-more');
+  newButton.innerHTML = 'Click to See Below!';
+  newButton.style.margin = '2%';
```
Comment (bitsagarob) [inline Week1/homework/main.js:212]: you can set classes from you JS and set your styling in css files, separation makes it cleaner

**Review by bitsagarob — CHANGES_REQUESTED** [review]
good commenting but you can remove the unused old commented code, makes it cleaner

**Pull request closed without merging**